src/compiler.rs

```diff
@@ -324,8 +324,14 @@ impl AmberCompiler {
 
     #[cfg(not(windows))]
     fn find_bash() -> Option<Command> {
-        let mut command = Command::new("/usr/bin/env");
-        command.arg("bash");
-        Some(command)
+        if env::var("GITHUB_ACTIONS_BASH_CONTAINER").is_ok() {
+            let mut command = Command::new("/usr/bin/docker");
```

Comment:
Can we rely on the `docker` executable existing here? Would it be better to pick it up from the `$PATH` variable, or would this pose a security risk of some sort?

Reply:
Replaced with `"docker"`. 595ca535a80f26da607bf921b504807f6e5a7a8a
I am not an expert, but I cannot see any security risk from here.